packages/realm-common/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "./dist/bundle.cjs.js",
   "module": "./dist/bundle.es.js",
   "types": "./dist/bundle.d.ts",
+  "react-native": "./dist/bundle.dom.es.js",
```

Comment:
Seems odd that react-native would source `dom` file.  Does this just happen to work because they are not using conflicting modules?

Reply:
Well - actually. Thinking more about this, I could use "browser" here instead.